cuda/base/cusparse_bindings.hpp

```diff
@@ -44,9 +44,29 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 namespace gko {
+/**
+ * @brief The device specific kernels.
+ *
+ * @ingroup kernels
+ */
 namespace kernels {
+/**
+ * @brief The CUDA namespace.
+ *
+ * @ingroup cuda
+ */
 namespace cuda {
+/**
+ * @brief The CUSPARSE namespace.
+ *
+ * @ingroup cusparse
+ */
 namespace cusparse {
+/**
+ * @brief The detail namespace.
```

Comment:
Do you need to document this? Also, it's not written as `@internal` here though it is way below in one of the `core` documentation of your PR.

Reply:
This file is not in the usr documentation anyway. So, the detail namespace would be displayed only in the dev doc. But I think if I remove the doc for the detail namespace, because the detail namespace is in the cusparse namespace, it disappears, so I had to leave it.

Reply:
OK.